### PR TITLE
Adding modal for each person

### PR DESCRIPTION
**Pull request by MostafaBalata**
screen shot 2017-08-17 at 3 43 46 pm

### CODE REVIEW

**Comment by MostafaBalata** [comment]
screen shot 2017-08-17 at 4 57 25 pm

File: package.json [hunk 1]
```diff
@@ -50,6 +50,7 @@
     "build": "NODE_ENV=production bash -c 'gulp build && webpack'"
   },
   "dependencies": {
+    "@types/vex-js": "^2.3.29",
```
Comment (krzkaczor) [inline package.json]: remove it since we dont use it. `yarn delete` oterwise you skip changes to yarn.lock

File: page/sass/common.scss [hunk 2]
```diff
@@ -53,4 +53,8 @@
         background: white;
         color: $blue-darker;        
     }
+}
+
+[id^=team_member_], [id^=advisor_] {
```
Comment (krzkaczor) [inline page/sass/common.scss]: please use less hardcore selector :D Attach there additional class for example.

File: page/sass/vex-modal.scss [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    a.close-modal{
+        background: transparent;
+        border: 0;
+        color: black;
+        cursor: pointer;
+        display: block;
+        height: 44px;
+        margin: 0;
+        outline: 0;
+        overflow: visible;
```
Comment (krzkaczor) [inline page/sass/vex-modal.scss]: i think the best solution would be to create our own theme. But let it be.

File: page/ts/index.ts [hunk 4]
```diff
@@ -1,8 +1,41 @@
...
+import "!style-loader!css-loader!vex-js/dist/css/vex.css";
+import "!style-loader!css-loader!vex-js/dist/css/vex-theme-os.css";
+
+vex.defaultOptions.className = 'vex-theme-os'
+vex.registerPlugin(vexDialog)
+
+const getPersonModal: any = function(name: string, image: string, preTitle:string, title: string, bio:string){
+    return {
+        input: [
+            '<a href="#" class="close-modal"></a>',
```
Comment (krzkaczor) [inline page/ts/index.ts]: use es6 multineline string `

File: page/ts/index.ts [hunk 5]
```diff
@@ -11,12 +44,20 @@ $(document).ready(function(){
             dots: true,
             autoPlay: 3000, //Set AutoPlay to 3 seconds
         }
-
     );
 
     const seeMore: string = "+ More";
     const seeLess: string = "- Less"; 
-    
+    $('[id^=team_member_], [id^=advisor_]').click(function(){
```
Comment (krzkaczor) [inline page/ts/index.ts]: can't u just add some class to make this selector easier?

File: page/blocks/person.ejs [hunk 6]
```diff
@@ -1,9 +1,13 @@
...
     <div class="person-info">
         <img class="rounded-image" src="<%= image %>"/>
         <div class="person-details">
-            <h4 class="name"><a href="#" ><%= name %></a></h4>
-            <h4 ><%= title %></h4>
+            <h4 class="name"><a href="#" > <%= name %></a></h4>
+            <h4 class="title"><%= title %></h4>
         </div>
     </div>
+    <p class="bio hidden">
```
Comment (krzkaczor) [inline page/blocks/person.ejs]: Idont like this approach at all. This uses html markup only to store data that later we will read it via jquery. Lets revisit it when we will have all data in yml.

Comment (MostafaBalata) [inline page/blocks/person.ejs]: ok

**Pull request merged**